### ISSUE TITLE
1.8 Issue in 3.0

### ISSUE THREAD

**Issue by landoncrabtree** [Iridium-Development/IridiumSkyblock#243]
I'm not sure if this has been reported before, but when using the recode, it seems the config.yml is always generated using newer Spigot versions.

For example, the prefix is a gradient, sounds are ENTITY_PLAYER_LEVELUP, etc. Console throws a lot of errors about materials. And if you modify the config to make it 1.8 supported, it just gets reset on the next server start.

The only way I have been able to use IridiumSkyblock on 1.8 without errors is using 2.6.7.

**Comment by PeachesMLG** [comment]
Pro tip when mentioning "loads of errors in consoles" its usually a good idea to provide those errors

**Comment by landoncrabtree** [comment]
Sorry about that. I created this issue after I already switched to 2.6.7, so console was no longer producing errors. I've switched back temporarily to the latest version (3.0.6) so I can produce the errors.

When creating an island (after selecting the schematic):
https://paste.landon.pw/raw/uwonerud

And then additionally, the fact that any modifications to the configuration file get wiped once the server is restarted. For example, I modified the following:

<GRADIENT:09C6F9>&lIridiumSkyblock</GRADIENT:045DE9> &8»  -> &b&lSKYBLOCK &7
ENTITY_PLAYER_LEVELUP -> LEVEL_UP

Running /is reload saves the changes, but once the server is restarted, those changes are lost. 

Thanks,
Landon
